### ISSUE TITLE
регистрируюсь

### ISSUE THREAD

**Issue by Artem1000I** [jenkins-infra/helpdesk#3342]
### Type(s) of account issue

I've been blocked by the anti-spam system

### Summary

Регестрируюсь для учёбы

### jenkins.io account username

filolog999333

### jenkins.io account email

filolog999333@mail.ru

### Which URL are you trying to login at?

https://www.jenkins.io/download/thank-you-downloading-windows-installer/

**Comment by dduportal** [comment]
Hi @Artem1000I , this issue seems a duplicate of #3343. Can you explain why do you need 2 accounts please?

**Comment by dduportal** [comment]
Closing as duplicate of #3343
